airbyte_cdk/sources/declarative/decoders/__init__.py

```diff
@@ -3,9 +3,9 @@
 #
 
 from airbyte_cdk.sources.declarative.decoders.decoder import Decoder
-from airbyte_cdk.sources.declarative.decoders.json_decoder import JsonDecoder, JsonlDecoder, IterableDecoder
+from airbyte_cdk.sources.declarative.decoders.json_decoder import JsonDecoder, JsonlDecoder, IterableDecoder, GzipJsonDecoder
```

Comment:
nit: shouldn't those imports be in alphabetical order?

Reply:
I guess this will be fixed later

https://github.com/airbytehq/airbyte-python-cdk/blob/1e0f777924a902e6e8d2909992459d1188c9e6e6/pyproject.toml#L105

Reply:
Ok, that explains why. Thanks for the context! I'll resolve this comment